docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -458,6 +397,8 @@ sudo python3 plant-watering-robot.py
 ```
 
 To tinker this example code to work best for you, determine at what [analog value from the soil moisture readings](#test-your-soil-moisture-readings-on-your-pi) you want to water your plant, as your thirsty plant's average moisture reading might differ from our example value of `60000`.
+
+Test your sensor by putting it in air, water, and different soils to see how the values change to determine your baseline for wet and dry values.
```

Comment:
```suggestion
Test your sensor by putting it in air, water, and different soils to see how the values change and determine your baseline for wet and dry values.
```
Instead of “to” twice